src/components/ButtonPanel.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from './Button';
+import './ButtonPanel.css';
+
+const arraySymbols = [
+  ['AC', '+/-', '%', '÷'],
+  ['7', '8', '9', 'X'],
+  ['4', '5', '6', '-'],
+  ['1', '2', '3', '+'],
+  ['0', '.', '='],
+];
+
+const buttonOperation = (item) => {
+  if (item === '÷' || item === 'X' || item === '-' || item === '+' || item === '=') {
+    return true;
+  }
+  return false;
+};
```

Comment:
[Optional]

We could also do this :thinking: 

```js
const buttonOperation = (item) => ['÷', 'X', '-', '+', '='].includes(item)
```

Reply:
nice!